volume/devicemapper/devmapper/deviceset.go

```diff
@@ -1127,7 +1129,9 @@ func (devices *DeviceSet) checkGrowBaseDeviceFS(info *devInfo) error {
 	}
 
 	if devices.baseFsSize < devices.getBaseDeviceSize() {
-		return fmt.Errorf("devmapper: Base device size cannot be smaller than %s", units.HumanSize(float64(devices.getBaseDeviceSize())))
+		return fmt.Errorf(
+			"devmapper: Base device size cannot be smaller than %s",
+			units.CustomSize("%.4g%s", float64(devices.getBaseDeviceSize()), 1024.0, sizeAbbrs))
```

Comment:
This, actually: https://github.com/moby/moby/blob/05ad14fc1bc5626895a419a991fcf8932eb91d27/daemon/graphdriver/devmapper/deviceset.go#L2648
and
https://github.com/moby/moby/blob/05ad14fc1bc5626895a419a991fcf8932eb91d27/daemon/graphdriver/devmapper/deviceset.go#L40

It's RAMInBytes, so you're good.